scripts/rnnlm/train_rnnlm.sh

```diff
@@ -175,14 +177,19 @@ while [ $x -lt $num_iters ]; do
         else dest_number=$[x+1]; fi
         # in the normal case $repeated data will be just one copy.
         repeated_data=$(for n in $(seq $num_repeats); do echo -n $dir/text/$split.txt ''; done)
-
+        
+        rnnlm_l2_factor=$(perl -e "print (1.0/$this_num_jobs);")
+        embedding_l2_factor=$rnnlm_l2_factor
 
         # Run the training job or jobs.
         $cmd $queue_gpu_opt $dir/log/train.$x.$n.log \
            rnnlm-train \
              --rnnlm.max-param-change=$rnnlm_max_change \
+             --rnnlm.l2_regularize_factor=$rnnlm_l2_factor \
              --embedding.max-param-change=$embedding_max_change \
              --embedding.learning-rate=$embedding_lrate \
+             --embedding.l2_regularize=$embedding_l2 \
+             --embedding.l2_regularize_factor=$embedding_l2_factor \
```

Comment:
I don't think it's necessary to have two separate options here-- just have a single l2-regularize option and compute it at the script level.  Specifying the factor was only necessary for the model-level one because the component-level l2 values can be different.

Reply:
Ok, will remove the embedding l2 factor.